packages/eslint-plugin-fiori-tools/package.json

```diff
@@ -25,8 +25,8 @@
     },
```

Comment:
@devinea 
The devDependency `eslint@7.32.0` has a depedency to `minimatch@3.0.4` which has a security issue. It looks like there is no newer `7.x` version with a fix. Could we switch to `8.x` e.g. `8.57.0`?

Reply:
Yeah makes sense.

Reply:
Thanks